### PR TITLE
translations have been removed from symfony.com

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #4078 |

The old page will be kept for people following a deep link or a bookmark, but the actual content has been removed and is replaced with a link to [the blog article](http://symfony.com/blog/discontinuing-the-symfony-community-translations).

### CODE REVIEW

**Comment by javiereguiluz** [comment]
:+1: it makes sense to remove this article.

**Comment by WouterJ** [comment]
I think we can just completely remove this article. When people find the 404 page and start looking, they'll soon find the blog post.

Also, there doesn't seem to be [any backlinks for this article](https://www.google.nl/search?q=link%3Ahttp%3A%2F%2Fsymfony.com%2Fdoc%2Fcurrent%2Fcontributing%2Fdocumentation%2Ftranslations.html).

**Comment by xabbuh** [comment]
I'm fine with that either.

@javiereguiluz @weaverryan What do you think?

**Comment by javiereguiluz** [comment]
In my previous comment I was referring to remove the content but not the article. I'd prefer not to show a 404 error, even if the traffic to this article is marginal.

File: contributing/documentation/translations.rst [hunk 1]
```diff
@@ -1,87 +1,6 @@
 Translations
 ============
 
-The Symfony documentation is written in English and many people are involved
-in the translation process.
+Featuring translations of the Symfony documentation `has been terminated`_.
```
Comment (weaverryan) [inline contributing/documentation/translations.rst]: This seems a bit harsh, and I know some people are sensitive about this decision. Something like this:

>
The Symfony documentation is not officially translated, though some community groups still
maintain some translations. For more information, see <link>.

**Comment by weaverryan** [comment]
:+1: for this approach

File: contributing/documentation/translations.rst (hunk 1 above)
Comment (xabbuh) [inline contributing/documentation/translations.rst]: Yeah, that's probably a bit nicer. :)

**Pull request merged**